### PR TITLE
Add nodemon for automatically restart after any code change

### PR DESCRIPTION
**Pull request by Pariyawit**
I think this might be useful during development. However, if a script is changed and restarted then data like botInfo won't be saved too...

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,9 +1,2325 @@
 {
   "name": "twitch_tools",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
```
Comment (thananon) [inline package-lock.json:6]: is this 3000+ lines change necessary?

Comment (Pariyawit) [reply package-lock.json:6]: Hi. Looks like these are dependencies from nodemon.

I ran `npm audit --json` and nodemon seems to add 117 dependencies.
```
{
  "auditReportVersion": 2,
  "vulnerabilities": {},
  "metadata": {
    "vulnerabilities": {
      "info": 0,
      "low": 0,
      "moderate": 0,
      "high": 0,
      "critical": 0,
      "total": 0
    },
    "dependencies": {
      "prod": 114,
      "dev": 117,
      "optional": 1,
      "peer": 0,
      "peerOptional": 0,
      "total": 230
    }
  }
}
```

File: package.json [hunk 2]
```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "cd 9armbot && nodemon 9armbot.js --ignore '*.json'"
+    "start": "cd 9armbot && node 9armbot.js",
+    "dev": "cd 9armbot && nodemon 9armbot.js --ignore '*.json'"
```
Comment (Pariyawit) [inline package.json:9]: I added seperate command. now anyone can run with `npm run dev` instead

**Comment by kaozaza2** [comment]
I think `package-lock.json` should in `.gitignore` file ><

**Comment by thananon** [comment]
I am worried about adding a 3000 line `package-lock.json` into a project with ~500 lines. Is the file necessary or it can be safely ignored?

**Comment by thananon** [comment]
@Aitthi Let me know what you think. The only problem is with package-lock.json. If you think its ok, I will merge.

**Comment by Aitthi** [comment]
I think there is no problem. But this has been ignored.

**Review by Aitthi — COMMENTED** [review]
I think there is no problem. But this has been ignored.

**Comment by Pariyawit** [comment]
I've removed package-lock.json. sorry for any inconvenience.

**Pull request merged**